src/assets/scripts/client/airline/AirlineModel.js

```diff
@@ -207,40 +196,15 @@ export default class AirlineModel extends BaseModel {
 
     // TODO: the logic here can be simplified.
     /**
-     * Create a flight number/identifier
```

Comment:
well we still need the content of the docblock here too unless it no longer applies